core/src/main/scala/kafka/tools/ConsumerPerformance.scala

```diff
@@ -302,6 +305,11 @@ object ConsumerPerformance extends LazyLogging {
     val printMetricsOpt = parser.accepts("print-metrics", "Print out the metrics. This only applies to new consumer.")
     val showDetailedStatsOpt = parser.accepts("show-detailed-stats", "If set, stats are reported for each reporting " +
       "interval as configured by reporting-interval")
+    val pollLoopTimeoutOpt = parser.accepts("timeout", "Consumer polling loop timeout.")
```

Comment:
Can we make this more descriptive? How about this?
> The maximum allowed time between returned records

Also, maybe we can change the name to `recordFetchTimeout` or something like that?

Reply:
Could we also mention that the timeout is in milliseconds? And perhaps also name the variable `recordFetchTimeoutMs`.

Reply:
@dhruvilshah3
@hachikuji 

Hello, guys. Sure, I'll take care of that today. 

Would you like to apply changes for both the variable name and the UI parameter? If yes, then KIP should be updated too. 

Thanks for closing the vote session. Been away for a while, I have more time now.